panther_lights/README.md

```diff
@@ -69,51 +75,51 @@ Node responsible for scheduling animations displayed on LED panels based on the
 - `timeout` [*float*, default: **120.0**]: time in seconds after which animation will be removed from the queue.
 
+Default animations can be found in the table below:
+
+| ID  | NAME              | PRIORITY | DESCRIPTION                                                 |
+| :-: | ----------------- | :------: | ----------------------------------------------------------- |
+| 0   | E_STOP            | 1        | red expanding from the center to the edges                  |
+| 1   | READY             | 3        | green expanding from center to the edges                    |
+| 2   | ERROR             | 1        | red blinking twice                                          |
+| 3   | MANUAL_ACTION     | 3        | blue expanding from the center to the edges                 |
+| 4   | AUTONOMOUS_ACTION | 3        | orange expanding from center to the edges                   |
+| 5   | GOAL_ACHIEVED     | 2        | purple blinking three times                                 |
+| 6   | LOW_BATTERY       | 1        | two orange stripes moving towards the center, repeats twice |
+| 7   | CRITICAL_BATTERY  | 1        | two red stripes moving towards the center, repeats twice    |
+| 8   | BATTERY_STATE     | 3        | two stripes moving towards the edges stopping at a point representing battery percentage and filling back to the center, color changes from red to green |
+| 9   | CHARGING_BATTERY  | 1        | solid color with a duty cycle proportional to the battery percentage, color changes from red to green |
+
 ### Animation types
 
 #### Animation
 
-Basic animation definition. Supported keys are:
+Basic animation definition. Keys are inherited from basic **Animation** class by all animations. Supported keys are:
 
-- `brightness` [*float*, optional]: animation brightness. This will overwrite `global_brightness` for a given animation.
-- `duration` [*float*]: duration of a single image animation.
-- `repeat` [*int*, optional]: number of times the animation will be repeated, by default animation will run once.
-- `type` [*string*]: required field specyfying animation type, currently suported animation types are: `image_animation`, `battery_animation`, `charging_animation`.
+- `brightness` [*float*, optional]: animation brightness relative to APA102 driver `global_brightness`. Range between [0,1].
+- `duration` [*float*, required]: duration of a single image animation.
+- `repeat` [*int*, default: **1**]: number of times the animation will be repeated, by default animation will run once.
+- `type` [*string*, required]: required field specyfying animation type, currently suported animation types are: `image_animation`, `battery_animation`, `charging_animation`.
 
-:bulb: **NOTE:** The overall display duration of an animation is a product of a single image duration and repeat count. It can't exceed 10 seconds.
+> **NOTE:**: The overall display duration of an animation is a product of a single image duration and repeat count. It can't exceed 10 seconds.
```

Comment:
```suggestion
> **NOTE:**: Overall display duration of an animation is a product of a single image duration and repeat count. Result of `duration` x `repeat`  can't exceed 10 seconds. If animation fails to fulfill the requirement it will be omited while loading.
```